src/main/java/redis/clients/jedis/util/CaffeineCSC.java

```diff
@@ -0,0 +1,95 @@
+package redis.clients.jedis.util;
```

Comment:
Can we subpackage the caching bits to `jedis.util.caching` or similar?

Reply:
I guess... but these are just two classes.
There are many other packages that make sense. If we continue having new packages for one or two classes, we will end up with having way too many.

Reply:
Fair - but these are at least related. util is a catch-all, a dumping ground (not yours, generally). It wouldn't be bad to bring some order to this.